tornado/httputil.py

```diff
@@ -243,6 +244,86 @@ def __deepcopy__(self, memo_dict):
         return self.copy()
 
 
+_cookie_value_decode = Cookie.SimpleCookie().value_decode
+_cookie_attr_reserved = Cookie.Morsel._reserved
+_cookie_attr_flags = set(["secure", "httponly"])
+
+# part of Cookie library module
+# https://docs.python.org/2.7/library/cookie.html
```

Comment:
No difference. This code duplication needs only for partial (key=value) custom parse cookies with catch exceptions if need, because stdlib not provide convenient method to implement some custom handle other situation. But problem with incorrect name is very frequent in real world, even if some symbols is not allowed in RFC 2616 (http://tools.ietf.org/html/rfc2616#section-2.2). Probably solution is not pretty but it works. I will be glad if you offer anything better :)